### PR TITLE
Set up a basic block type

### LINKED ISSUE #12: Set up a block type
Basic block type with content field

### PR DESCRIPTION
**Pull request by tannguyen04**
## Description

closes #12

### CODE REVIEW

**Comment by tannguyen04** [comment]
Hi @simesy 
I have setup a basic block type with content field.
https://setup-basic-block-type-2lzuyga-fw7ucnhzydcsy.au.platformsh.site/ce

File: config/sync/core.entity_view_display.block_content.basic.default.yml [hunk 1]
```diff
@@ -4,11 +4,21 @@ status: true
...
   default_config_hash: hBNNDTFwakREOTa6GGMqN899Iyrii0hInwSJtQ7Kj30
 id: block_content.basic.default
 targetEntityType: block_content
 bundle: basic
 mode: default
-content: {  }
+content:
+  body:
+    type: text_default
+    label: above
```
Comment (simesy) [inline config/sync/core.entity_view_display.block_content.basic.default.yml:19]: Should not be a label?

File: config/sync/field.field.block_content.basic.body.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.basic
+    - field.storage.block_content.body
+  module:
+    - text
+id: block_content.basic.body
+field_name: body
```
Comment (simesy) [inline config/sync/field.field.block_content.basic.body.yml:11]: In the node the field is called `markup` I would like to keep this consistent.

**Review by simesy — CHANGES_REQUESTED** [review]

**Comment by simesy** [comment]
@tannguyen04 can you also add a block in the footer with 

```
Free Sauce continues to acknowledge the Traditional Owners and Custodians of the lands on which we work. We pay our respects to Indigenous Elders past, present, and emerging. Sovereignty has never been ceded. It always was and always will be Aboriginal land.
```

I'll add other stuff later.

**Comment by tannguyen04** [comment]
> @tannguyen04 can you also add a block in the footer with
> 
> ```
> Free Sauce continues to acknowledge the Traditional Owners and Custodians of the lands on which we work. We pay our respects to Indigenous Elders past, present, and emerging. Sovereignty has never been ceded. It always was and always will be Aboriginal land.
> ```
> 
> I'll add other stuff later.

Added https://setup-basic-block-type-2lzuyga-fw7ucnhzydcsy.au.platformsh.site/ce

**Review by simesy — APPROVED** [review]

**Pull request merged**